source/rust_verify/src/rust_to_vir_base.rs

```diff
@@ -534,15 +534,36 @@ pub(crate) fn get_impl_paths_for_clauses<'tcx>(
                                     }
                                     _ => {}
                                 }
+                            } else if Some(trait_def_id) == tcx.lang_items().sized_trait()
+                                || Some(trait_def_id) == tcx.lang_items().tuple_trait()
+                                || Some(trait_def_id) == tcx.lang_items().pointee_trait()
+                                || Some(trait_def_id) == tcx.lang_items().sync_trait()
+                                || matches!(
+                                    verus_items::get_rust_item(tcx, trait_def_id),
+                                    Some(RustItem::Send | RustItem::Thin)
+                                )
+                            {
+                                // Sized, Tuple, Pointee, Thin are all ok to do nothing.
+                                // There can't be user impls of these traits, they can only be built-in.
+
+                                // TODO: Send and Sync needs handling, or a rigorous argument why it's ok to skip;
+                                // See https://github.com/verus-lang/verus/issues/1335
```

Comment:
I commented on https://github.com/verus-lang/verus/issues/1335 .  Is this TODO considered resolved, or does it need more discussion?

Reply:
I'm genuinely unsure about this one; if there's no issue, I'd still like to see a rigorous argument.

Reply:
Ok, I tried to write out a more rigorous argument in https://github.com/verus-lang/verus/issues/1335 .